vcstool/clients/git.py

```diff
@@ -281,24 +284,26 @@ def import_(self, command):
             'returncode': 0
         }
 
-    def _get_url(self):
+    def _get_remote_urls(self):
         cmd_remote = [GitClient._executable, 'remote', 'show']
         result_remote = self._run_command(cmd_remote)
         if result_remote['returncode']:
-            result_remote['output'] = 'Could not determine remote: ' + \
+            result_remote['output'] = 'Could not determine remotes: ' + \
                 result_remote['output']
-            return result_remote
-        remote = result_remote['output']
-        result_url = self._get_remote_url(remote)
-        if result_url['returncode']:
-            return result_url
-        url = result_url['output']
-        return {
-            'cmd': ' && '.join([result_remote['cmd'], result_url['cmd']]),
-            'cwd': self.path,
-            'output': [url, remote],
-            'returncode': 0
-        }
+            return [result_remote]
+        remote_urls = []
+        for remote in result_remote['output'].splitlines():
+            result_url = self._get_remote_url(remote)
+            url = 'n/a' if result_url['returncode'] else result_url['output']
+            remote_urls.append(
+                {
+                    'cmd': ' && '.join([result_remote['cmd'],
+                                        result_url['cmd']]),
+                    'cwd': self.path,
+                    'output': [url, remote],
+                    'returncode': result_url['returncode']
+                })
+        return remote_urls
```

Comment:
Maybe it would be easier to return only a result rather than a list of results? The `output` of the single result could still be a list of url-remote-tuples. This would make the error handling in the calling location much easier.